docs/content/guides/developer/app-examples/blackjack.mdx

```diff
@@ -98,56 +98,56 @@ The [`page` component](https://github.com/MystenLabs/blackjack-sui/blob/main/app
+- **Randomness generation:** Fair game outcomes are also ensured by the [`CollectMouseRandomness`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/CollectMouseRandomness.tsx) component. This component is using the [`useMouseRandomness`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/hooks/useMouseRandomness.ts) custom hook, and is in charge of capturing some user's mouse movements and generating a random bytes array. This array is converted to a hexadecimal string (`randomness`) and used in the create-game transaction.
 
-- **Card Displaying and Management:** The [DealerCards](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/DealerCards.tsx) and the [PlayerCards](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/PlayerCards.tsx) components are used to display the total points and the cards owned by the dealer and the player respectively.
+- **Card displaying and management:** The [`DealerCards`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/DealerCards.tsx) and the [`PlayerCards`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/PlayerCards.tsx) components are used to display the total points and the cards owned by the dealer and the player respectively.
 
-- **Game Actions:** The [GameActions](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/GameActions.tsx) component is used to display the Hit and Stand buttons, and trigger the corresponding actions, as they are exported by the [useBlackjackGame](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/hooks/useBlackjackGame.ts) hook to execute the corresponding transactions.
+- **Game actions:** The [`GameActions`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/GameActions.tsx) component is used to display the **Hit** and **Stand** buttons, and trigger the corresponding actions, as they are exported by the [`useBlackjackGame`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/hooks/useBlackjackGame.ts) hook to execute the corresponding transactions.
 
-- **BlackjackBanner**: The [BlackjackBanner](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/BlackjackBanner.tsx) component is used as a custom view to display when the player wins with a blackjack.
+- **`BlackjackBanner`**: The [`BlackjackBanner`](https://github.com/MystenLabs/blackjack-sui/blob/main/app/src/components/home/BlackjackBanner.tsx) component is used as a custom view to display when the player wins with a blackjack.
 
-## Comparison: Blackjack and Coin Flip
+## Comparison: Blackjack and coin flip
```

Comment:
Name of example. I'd argue Blackjack and Coin Flip examples demo a blackjack game and digital coin flip.